tests/integ/test_stored_procedure_profiler.py

```diff
@@ -283,18 +277,15 @@ def oom_sp(session: snowflake.snowpark.Session) -> str:
 
     profiler_session.stored_procedure_profiler.set_active_profiler("LINE")
 
-    with pytest.raises(
-        SnowparkSQLException, match="Function available memory exhausted"
-    ):
+    with pytest.raises(SnowparkSQLException, match="fake out of memory") as err:
         profiler_session.call(function_name)
-    res = profiler_session.stored_procedure_profiler.get_output()
+        id = profiler_session.stored_procedure_profiler._get_last_query_id()
+        assert id in str(err)
 
     profiler_session.stored_procedure_profiler.disable()
 
     profiler_session.stored_procedure_profiler.register_modules()
 
-    assert res is not None and "oom_sp" in res
-
```

Comment:
why are we removing this?

Reply:
hmm then I don't think we are truly integ testing profiler get_output? Do we have any other tests for get_output elsewhere in this file?
If not, lets try to make the same test more reliable like an even smaller `PYTHON_UDF_CGROUP_MEMSIZE`. Maybe discuss with udf team if they are some more reliable tests in their test suites.